src/components/GDPRForm/index.tsx

```diff
@@ -5,11 +5,11 @@ export const GDPRForm = () => (
     <>
         <br />
         <p className="px-4 font-bold text-center z-10 relative mb-0 text-sm md:text-lg">
-            Want PostHog hosted on an EU Cloud?
+            Interested in PostHog Cloud EU?
```

Comment:
```suggestion
            PostHog Cloud EU. A fully managed deployment, hosted in the EU.
```

My copy isn't _great_. But we need to bear in mind the GDPRForm element is used across blog posts too, where we don't actually mention or describe what PostHog Cloud EU is -- so just saying 'Interested in it?' isn't going to be compelling.